todo_app/trello_api_calls.py

```diff
@@ -0,0 +1,61 @@
+import os, requests
+
+class Item:
+    def __init__(self,title,id,status):
+        self.title = title
+        self.status = status
+        self.id = id
+
+    def create_card(title, list_category):
+        list_id = get_list_id(list_category)
+        payload = {'key':os.environ.get('TRELLO_KEY'),'token':os.environ.get('TRELLO_TOKEN'), \
+        'idList':list_id}
+        url = api_prefix + '1/cards'
+        card_title = {"name": title}
+        api_response = requests.post(url, params=payload, data = card_title)
+        api_response_list = api_response.json()
+        return Item(api_response_list["name"],api_response_list["id"], list_category)
+
+api_prefix = 'https://api.trello.com/'
+
+def call_api(api_suffix):
+    payload = {'fields':'name','key':os.environ.get('TRELLO_KEY'),'token':os.environ.get('TRELLO_TOKEN')}
+    url = api_prefix + api_suffix
+    api_response = requests.get(url, params=payload)
+    api_response_list = api_response.json()
+    return api_response_list
+
+def get_board_id():
+    api_suffix_get_board = '1/members/' + os.environ.get('TRELLO_USER_NAME')+ '/boards'
```

Comment:
No harm in doing it this way, but for simplicity worth noting that you can hardcode 'me' in place of the username and Trello will work out the user based on the api key

It's worth also noting that the user now needs to provide a board name as an environment variable; it's not quite as user friendly but it might simplify this bit if we asked them to set the board id as an environment variable explicitly and then we could skip this stage. This isn't necessary, but is worth thinking about the trade offs